### PR TITLE
Junction tables

### PR DESCRIPTION
**Pull request by yegonkimutai**
In this Pull Request, I did the following ;

- [ ] Create the vets table and added data.
- [ ] Create specializations junction table to link species and vets tables.
- [ ] Create visits junction table to link animals and vets tables.
      
 Run the following queries to confirm insertion of data:
- [ ] Animal last seen by William Tatcher

Screenshot (56)

- [ ] Total animals visited by Stephanie Mendez

Screenshot (57)

- [ ] List of all vets and their specializations including no specialization

Screenshot (58)

- [ ] List of animals visited by Stephanie Mendez between April 1st and August 30th, 2020

Screenshot (66)

- [ ] Animal with the most visits

Screenshot (60)

- [ ] First animal that visited Maisy Smith

Screenshot (61)

- [ ] All details of the most recent visit

Screenshot (64)

- [ ] Number of visits made by a non-specialized vet

Screenshot (62)

- [ ] Specialization Maisy should consider getting

Screenshot (63)

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -62,4 +62,43 @@ UPDATE animals SET owners_id = 3 WHERE name IN('Devimon', 'Plantmon');
 UPDATE animals SET owners_id = 4 WHERE name IN('Charmander', 'Squirtle','Blossom');
 UPDATE animals SET owners_id = 5 WHERE name IN('Angemon', 'Boarmon');
 COMMIT;
-SELECT * FROM animals;
\ No newline at end of file
+SELECT * FROM animals;
+
+-- insert into vets
+INSERT INTO vets(name, age, graduation_date)
+VALUES('William Tatcher', 45, '2000-03-23'),
```
Comment (BregornOriginal) [inline data.sql]: - [ ] Kindly correct the date of `William Tatcher` to match with the provided `2000-04-23`

File: data.sql [hunk 2]
```diff
@@ -62,4 +62,43 @@ UPDATE animals SET owners_id = 3 WHERE name IN('Devimon', 'Plantmon');
...
+INSERT INTO specializations(species_id, vet_id)
+VALUES((SELECT id FROM species WHERE name = 'Pokemon'),(SELECT id FROM vets WHERE name = 'William Tatcher')),
+((SELECT id FROM species WHERE name = 'Pokemon'),(SELECT id FROM vets WHERE name = 'Stephanie Mendez')),
+((SELECT id FROM species WHERE name = 'Digimon'),(SELECT id FROM vets WHERE name = 'Stephanie Mendez')),
+((SELECT id FROM species WHERE name = 'Digimon'),(SELECT id FROM vets WHERE name = 'Jack Harkness'))
+
+-- insert into visits
+INSERT INTO visits(animal_id, vet_id, date)
+VALUES((SELECT animalid FROM animals WHERE name = 'Augmon'), (SELECT id FROM vets WHERE name = 'William Tatcher'), '2020-05-24'),
+((SELECT animalid FROM animals WHERE name = 'Augmon'), (SELECT id FROM vets WHERE name = 'Stephanie Mendez'), '2020-07-22'),
```
Comment (BregornOriginal) [inline data.sql]: - [ ] Kindly fix the name of the pokemon to match with the provided which is `Agumon`

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
Hi @yegonkimutai  

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!

giphy (5)

## Highlights

✅ Linters are passing
✅ Good commit history
✅ Good PR title and description
✅ Correct use of Gitflow

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cynthiainga — APPROVED** [review]
Hi @yegonkimutai, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- [ ] All requested changes are implemented ✔️ 
- [ ] The query screenshots are present ✔️ 
- [ ] Correct Gitflow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can get the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**